src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/monetizationmodule/MonetizationModuleComponentObject.java

```diff
@@ -52,114 +58,137 @@ public void deleteCookieFromSearch() {
+	public void verifyAdsenseUnitNotShownInContent() {
+		waitForElementNotPresent(slotInContentAdsense);
+		PageObjectLogging.log("verifyAdsenseUnitNotShownInContent", "Adsense unit is not shown in content", true);
 	}
 
-	public void verifyMonetizationModuleShownBelowCategory() {
-		Assertion.assertTrue(checkIfElementOnPage(slotBelowCategory));
-		waitForElementByElement(slotBelowCategory);
-		PageObjectLogging.log("verifyMonetizationModuleSlotAboveTitleShown", "Monetization module is visible below category", true);
+	public void verifyAdsenseUnitShownBelowCategory() {
+		waitForElementByElementLocatedBy(slotBelowCategoryAdsense);
+		Assertion.assertTrue(checkIfElementOnPage(slotBelowCategoryAdsense));
+		Assertion.assertTrue(checkIfElementOnPage(adsenseInsOthers));
+		PageObjectLogging.log("verifyAdsenseUnitShownBelowCategory", "Adsense unit is visible below category", true);
 	}
 
-	public void verifyMonetizationModuleNotShownBelowCategory() {
-		Assertion.assertFalse(checkIfElementOnPage(slotBelowCategory));
-		PageObjectLogging.log("verifyMonetizationModuleSlotAboveTitleNotShown", "Monetization module is not shown below category", true);
+	public void verifyAdsenseUnitNotShownBelowCategory() {
+		waitForElementNotPresent(slotBelowCategoryAdsense);
+		PageObjectLogging.log("verifyAdsenseUnitNotShownBelowCategory", "Adsense unit is not shown below category", true);
 	}
 
-	public void verifyMonetizationModuleShownAboveFooter() {
-		Assertion.assertTrue(checkIfElementOnPage(slotAboveFooter));
-		waitForElementByElement(slotAboveFooter);
-		PageObjectLogging.log("verifyMonetizationModuleSlotAboveTitleShown", "Monetization module is visible above footer", true);
+	public void verifyAdsenseUnitShownAboveFooter() {
+		waitForElementByElementLocatedBy(slotAboveFooterAdsense);
+		Assertion.assertTrue(checkIfElementOnPage(slotAboveFooterAdsense));
+		Assertion.assertTrue(checkIfElementOnPage(adsenseInsOthers));
+		PageObjectLogging.log("verifyAdsenseUnitShownAboveFooter", "Adsense unit is visible above footer", true);
 	}
 
-	public void verifyMonetizationModuleNotShownAboveFooter() {
-		Assertion.assertFalse(checkIfElementOnPage(slotAboveFooter));
-		PageObjectLogging.log("verifyMonetizationModuleSlotAboveTitleNotShown", "Monetization module is not shown above footer", true);
+	public void verifyAdsenseUnitNotShownAboveFooter() {
+		waitForElementNotPresent(slotAboveFooterAdsense);
+		PageObjectLogging.log("verifyAdsenseUnitNotShownAboveFooter", "Adsense unit is not shown above footer", true);
+	}
+
+	public void verifyAdsenseUnitWidth(int expectedInContent, int expectedOthers) {
+		int width;
+		List<WebElement> listWebElements = driver.findElements(MonetizationModuleAdsenseListBy);
+		for (WebElement elem : listWebElements) {
+			String slotName = elem.getAttribute(attributeNameSlot);
+			width = elem.findElement(By.cssSelector(".adsbygoogle")).getSize().width;
```

Comment:
You could put the By selector or the selector string as a class (const) variable